fragments/k8s/cdk/README.md

```diff
@@ -24,10 +24,7 @@ knowledge. It is comprised of the following components and features:
 
 Installation has been automated via [conjure-up](http://conjure-up.io/):
 
-    sudo apt-add-repository ppa:juju/stable
-    sudo apt-add-repository ppa:conjure-up/next
-    sudo apt update
-    sudo apt install conjure-up
+    sudo snap install conjure-up --classic
```

Comment:
@marcoceppi can you test the snap install on your system to make sure the failed lxd bridge doesn't interfere?